Makefile

```diff
@@ -886,20 +888,31 @@ ifeq ($(ENABLE_VERBOSE),yes)
 	$(MV) $(@F) $(INSTALL_LIBDIR)/
 else
 	@echo "Installing symlink $(@F) into $(INSTALL_LIBDIR)/"
-	@$(SYMLINK) $(<F) $(@F)
+	@$(SYMLINK) $(INSTALL_LIBDIR)/$(<F) $(@F)
 	@$(MV) $(@F) $(INSTALL_LIBDIR)/
 endif
```

Comment:
@isuruf Can you briefly explain this change? Also, if you are going to make this change, shouldn't it also be applied to the verbose use case above?

Reply:
I think I understand this change a little better. It appears that you want the symlink `libblis.so` to use an absolute path, even though the library to which it is symlinked is in the same directory? Before:
```
lrwxrwxrwx 1 field dept      34 Sep  7 13:24 libblis.so -> libblis.so.1.0.0
-rw-r--r-- 1 field dept 2340168 Sep  7 13:24 libblis.so.1.0.0
```
After:
```
lrwxrwxrwx 1 field dept      34 Sep  7 13:24 libblis.so -> /home/field/blis/lib/libblis.so.1.0.0
-rw-r--r-- 1 field dept 2340168 Sep  7 13:24 libblis.so.1.0.0
```
But it's still not clear to me why you would want/need this.

Reply:
Absolute paths in shared library links is definitely not the norm on either Linux or macOS.

Reply:
I agree with @devinamatthews; unless there is a really good reason, I think we should stick with relative paths for symlinks. But I'm still open to hearing from @isuruf.